.github/workflows/buildCSharp.yml

```diff
@@ -7,33 +7,50 @@ on:
 jobs:
   build:
     name: "Build and release CSharp SDK"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
 
     steps:
       - name: "Checkout develop Branch"
         uses: actions/checkout@v2
         with:
           ref: develop
```

Comment:
I'm not sure why you insist on checking out develop?

Reply:
You do that on push to develop, already, so by default it would checkout develop.

Reply:
also, develop is the default branch, so even if you add a cron job to run nightly for eg, that's what it would use.

Reply:
This is just for me to remember that it needs to build on develop branch instead of master which I use in my other repos. As you pointed out it should work exactly the same without specifying it.